specs/wp-signup-spec.js

```diff
@@ -409,42 +446,198 @@ test.describe( `[${ host }] Sign Up  (${ screenSize }, ${ locale })`, function()
 						);
 					}
 					const paymentButtonText = await securePaymentComponent.paymentButtonText();
+					return assert(
+						paymentButtonText.includes( expectedCurrencySymbol ),
+						`The payment button text '${ paymentButtonText }' does not contain the expected currency symbol: '${ expectedCurrencySymbol }'`
+					);
+				}
+			);
+
+			test.it(
+				'Can then see the secure payment page with the expected products in the cart',
+				async function() {
+					const securePaymentComponent = new SecurePaymentComponent( driver );
+					const premiumPlanInCart = await securePaymentComponent.cartContainsProduct(
+						premiumPlanSlug
+					);
+					assert.equal( premiumPlanInCart, true, "The cart doesn't contain the premium plan" );
+					const numberOfProductsInCart = await securePaymentComponent.numberOfProductsInCart();
+					return assert.equal(
+						numberOfProductsInCart,
+						1,
+						"The cart doesn't contain the expected number of products"
+					);
+				}
+			);
+
+			test.it( 'Can submit test payment details', async function() {
+				const testCreditCardDetails = dataHelper.getTestCreditCardDetails();
+				const securePaymentComponent = new SecurePaymentComponent( driver );
+				await securePaymentComponent.enterTestCreditCardDetails( testCreditCardDetails );
+				await securePaymentComponent.submitPaymentDetails();
+				return await securePaymentComponent.waitForPageToDisappear();
+			} );
+
+			test.it( 'Can see the secure check out thank you page', async function() {
+				return await new CheckOutThankyouPage( driver ).displayed();
+			} );
+		}
+	);
+
+	test.describe(
+		'Sign up for a site on a personal paid plan coming in via /create as personal flow in GBP currency @parallel',
+		function() {
+			this.bailSuite( true );
+			const blogName = dataHelper.getNewBlogName();
+			const expectedBlogAddresses = dataHelper.getExpectedFreeAddresses( blogName );
+			const emailAddress = dataHelper.getEmailAddress( blogName, signupInboxId );
+			const password = config.get( 'passwordForNewTestSignUps' );
+			const currencyValue = 'GBP';
+			const expectedCurrencySymbol = '£';
+
+			test.before( async function() {
+				return await driverManager.ensureNotLoggedIn( driver );
+			} );
+
+			test.it( 'We can set the sandbox cookie for payments', async function() {
+				const sandboxCookieValue = config.get( 'storeSandboxCookieValue' );
```

Comment:
Same as the password variable. It could be extracted to the global scope.

Reply:
done 🐌